Dockerfile

```
@@ -5,12 +5,12 @@ FROM amazon/aws-lambda-python:$PY_VERSION
 # Declare it a second time so it's brought into this scope.
 ARG PY_VERSION=3.9
 # Get the base output file name from the command line if provided.
-ARG FILE_NAME=example
+ARG FILE_NAME=publish_egress_ip
```

I'm looking at making this non-project specific in the skeleton but leaving it as an argument to allow for customized release workflows etc. as an option. Thoughts?

```suggestion
ARG FILE_NAME=lambda_build
```

---

Good idea- I like that plan.  Are you ok with me leaving this as is for now and making that change later when it gets added to the skeleton?  Until we move to automated releases/deployments, I'd prefer to avoid having generic names for these zip files because it could lead to confusion.

---

Would you elaborate on why you think it could lead to confusion? I only ask because the `build` job already provides a different name in https://github.com/cisagov/publish-egress-ip-lambda/blob/35e58a7ac13f0d1805ae93415ce31d063b8a2239/.github/workflows/build.yml#L160 if you pull down the automatically build images.

---

> Would you elaborate on why you think it could lead to confusion?

If someone was building it locally and not overriding the `FILE_NAME` arg because they were being lazy. 😄
But I suppose my laziness is not a great reason, so I will make the change you suggested.

---

Just a note for future laziness you could always use a `docker-compose.override.yml` and add
```yaml
build:
  args:
    FILE_NAME: publish_egress_ip
```
there x)